Jellyfin.Plugin.KodiSyncQueue/Utils/ApiUserCheck.cs

```diff
@@ -1,22 +1,23 @@
-﻿using MediaBrowser.Controller.Entities;
-using MediaBrowser.Controller.Library;
+﻿using MediaBrowser.Controller.Library;
 using System.Collections.Generic;
+using Jellyfin.Data.Entities;
+using MediaBrowser.Controller.Entities;
 
 namespace Jellyfin.Plugin.KodiSyncQueue.Utils
 {
     public static class ApiUserCheck
     {
-        public static IEnumerable<T> TranslatePhysicalItemToUserLibrary<T>(T item, User user, ILibraryManager _libraryManager, bool includeIfNotFound = false)
+        public static IEnumerable<T> TranslatePhysicalItemToUserLibrary<T>(T item, User user, ILibraryManager libraryManager, bool includeIfNotFound = false)
```

Comment:
Looks like this function comes from https://github.com/jellyfin/jellyfin/blob/100e9d586dff877610c488936aa035f61f6fedb5/Emby.Server.Implementations/EntryPoints/LibraryChangedNotifier.cs#L441-L465